src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -164,11 +172,59 @@ export const thorchainSaversStakingOpportunitiesMetadataResolver = async ({
   }
 
   if (getConfig().REACT_APP_FEATURE_RUNEPOOL) {
+    const { data: reservePositions } = await axios.get<ThorchainRunepoolReservePositionsResponse>(
+      `${getConfig().REACT_APP_MIDGARD_URL}/member/thor1dheycdevq39qlkxs2a6wuuzyn4aqxhve4qxtxt`,
+    )
     const { data: runepoolInformation } =
       await axios.get<ThorchainRunepoolInformationResponseSuccess>(
         `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/runepool`,
       )
 
+    const { data: poolsResponse } = await axios.get<ThornodePoolResponse[]>(
+      `${getConfig().REACT_APP_THORCHAIN_NODE_URL}/lcd/thorchain/pools`,
+    )
+
+    const poolsByAsset = poolsResponse.reduce<Record<string, ThornodePoolResponse>>(
+      (acc, pool) => ({ ...acc, [pool.asset]: pool }),
+      {},
+    )
+
+    const underlyingAssetIds = reservePositions.pools.map(pool =>
+      poolAssetIdToAssetId(pool.pool),
+    ) as unknown as AssetIdsTuple
```

Comment:
q: Does this filter out the PoL-enabled assets we do not support?

Reply:
Added another check as we do upper in the file in [b4210de](https://github.com/shapeshift/web/pull/7459/commits/b4210de685c8e4e4f44ee4f62b647f0eef04b04a)

```
const underlyingAssetIds = reservePositions.pools
      .map(pool => poolAssetIdToAssetId(pool.pool))
      .filter(assetId => isSome(assetId) && opportunityIds.includes(assetId as OpportunityId))
```